src/Trace/TracerFactory.php

```diff
@@ -65,7 +65,7 @@ public static function getInstance(array $spanProcessors = []): self
 
     public function getTracer(string $name, string $version = ''): Tracer
     {
-        if ($this->tracers[$name] instanceof Tracer) {
+        if ($this->tracers != null && $this->tracers[$name] instanceof Tracer) {
```

Comment:
@beniamin - I had to change this implementation in the `TracerFactory` because PHP 7.4 was complaining about `Trying to access array offset on value of type null`

You okay with this implementation or do you have other ideas of how you'd like to handle this?